### DISCUSSION TITLE
수평으로 이미지 2개 있을 때, 본문 영역을 벗어나는 현상

### DISCUSSION THREAD

**Discussion by vg-rlo** [pronist/hELLO#82] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows
- 브라우저: edge, chrome
- 버전: 3.4.2 (6월 초 업데이트) 
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [e.g 3.4.0]

**설명**
[본문에서 이미지를 수평으로 2개 이어 붙여지는 기능을 사용할 때, 본문 영역 밖으로 벗어납니다.]

**발생 과정**
[화면 크기를 줄이면 본문영역이 줄어듬에 따라 이미지 크기가 같이 잘 줄어듭니다. 스킨이 본문 주변 공백이 없기 때문인거 같습니다. 하지만, 본문 영역이 넓은 경우 = 본문 좌우면에 여백 공간이 있는 경우에는 이미지가 벗어납니다.]

**스크린샷 및 링크**
화면 크기를 줄였을 때(정상): 
![image](https://user-images.githubusercontent.com/69677950/122500898-9ca40500-d02e-11eb-9b93-271e35b6e80b.png)

화면 크기가 클 때(버그 있음): ![image](https://user-images.githubusercontent.com/69677950/122500805-6ebec080-d02e-11eb-9938-ec15acd28acb.png)
해당 포스트 링크: https://vg-rlo.tistory.com/entry/%EA%B0%95%EC%9D%98%EB%85%B8%ED%8A%B8-Time-Series-Regression-Part-3

**Comment by pronist** [comment, marked as answer]
@vg-rlo 안녕하세요? 화면이 넓을때 이미지가 본문 바깥으로 나가는 것은 **의도된 것**입니다. **사용자가 이미지를 표현할 때 더 풍부하게 표현할 수 있도록 만들어 놓은 장치**입니다. 최대 1100px 까지 적용됩니다. 이미지를 붙여놓은 형태(그리드)에서 해당 장치를 끄는 옵션은 현재 없습니다. 이러한 장치는 **싱글이미지, 듀얼이미지, 슬라이드쇼에 적용**됩니다. 따라서 이것이 버그는 아닙니다.

제 포스트도 마찬가지랍니다 :)
<https://pronist.tistory.com/52>

**Comment by vg-rlo** [comment]
아핳..! 그렇군요 빠른 답변 감사합니다!
